src/DevChatter.Bot.Web/Startup.cs

```diff
@@ -32,7 +57,93 @@ public void ConfigureServices(IServiceCollection services)
+            services.AddSingleton<IClock, SystemClock>();
+
+            services.AddSingleton(typeof(ILogger<>), typeof(Logger<>));
+            services.AddSingleton(typeof(ILoggerAdapter<>), typeof(LoggerAdapter<>));
+
+            services.AddSingleton<IChatUserCollection, ChatUserCollection>();
+            services.AddSingleton<ICurrencyGenerator, CurrencyGenerator>();
+            services.AddSingleton<IAutomatedActionSystem, HangfireAutomationSystem>();
+            services.AddSingleton<ISettingsFactory, SettingsFactory>();
+            services.AddSingleton<IBotCommand, RouletteCommand>();
+            services.AddSingleton<RockPaperScissorsGame>();
+            services.AddSingleton<IBotCommand, RockPaperScissorsCommand>();
+            services.AddSingleton<IBotCommand, HeistCommand>();
+            services.AddSingleton<HeistGame>();
+            services.AddSingleton<IBotCommand, QuizCommand>();
+            services.AddSingleton<QuizGame>();
+
+            services.AddSingleton<HangmanGame>();
+            services.AddSingleton<IBotCommand, HangmanCommand>();
+            services.AddSingleton<IBotCommand, TopCommand>();
+            services.AddSingleton<IBotCommand, UptimeCommand>();
+            services.AddSingleton<IBotCommand, TaxCommand>();
+            services.AddSingleton<IBotCommand, GiveCommand>();
+            services.AddSingleton<IBotCommand, CoinsCommand>();
+            services.AddSingleton<IBotCommand, BonusCommand>();
+            services.AddSingleton<IBotCommand, StreamsCommand>();
+            services.AddSingleton<IBotCommand, ShoutOutCommand>();
+            services.AddSingleton<IBotCommand, QuoteCommand>();
+            services.AddSingleton<IBotCommand, AliasCommand>();
+            services.AddSingleton<IBotCommand, ScheduleCommand>();
+            services.AddSingleton(typeof(IList<>), typeof(List<>));
+            services.AddTransient(typeof(Lazy<>), typeof(Lazier<>));
+
+            services.AddSimpleCommandsFromRepository(repository);
+
+            services.AddSingleton<IBotCommand, HelpCommand>(); // TODO: make these work
+            services.AddSingleton<IBotCommand, CommandsCommand>(); // TODO: make these work
+
+            services.AddSingleton(provider => new CommandList(provider.GetServices<IBotCommand>().ToList()));
+
+            //builder.Register(ctx => new HelpCommand(ctx.Resolve<IRepository>()))
+            //    .OnActivated(e => e.Instance.AllCommands = e.Context.Resolve<CommandList>())
+            //    .AsImplementedInterfaces();
+            //builder.Register(ctx => new CommandsCommand(ctx.Resolve<IRepository>()))
+            //    .OnActivated(e => e.Instance.AllCommands = e.Context.Resolve<CommandList>())
+            //    .AsImplementedInterfaces();
+
+            //builder.Register(ctx => new CommandList(ctx.Resolve<IList<IBotCommand>>()))
+            //    .AsSelf()
+            //    .SingleInstance();
```

Comment:
Commented out code - is there any reason for it to stay in the code?

Reply:
We'd kept it through a bunch of commits until we replaced it. We should have deleted it after replacing it.